angular.json

```diff
@@ -18,8 +18,7 @@
             "polyfills": "src/polyfills.ts",
             "assets": [
               "src/assets",
-              "src/favicon.ico",
-              "src/manifest.json"
```

Comment:
If we don't need manifest.json why do we add one in this PR

Reply:
for some reason i dont see these lines in angular.json

Reply:
wait we need manifest.json for pwa

Reply:
hmm might have been lost in the merge, i'll add the line back